fabric-structure-api-v1/src/main/java/net/fabricmc/fabric/api/structure/v1/FabricStructurePool.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.structure.v1;
+
+import net.minecraft.structure.pool.StructurePool;
+import net.minecraft.structure.pool.StructurePoolElement;
+import net.minecraft.util.Identifier;
+
+/**
+ * Represents a modifiable structure pool that has several helper methods for modders.
+ */
+public interface FabricStructurePool {
+	/**
+	 * Adds a new {@link StructurePoolElement} to the {@link StructurePool}.
+	 * See the alternative {@link #addStructurePoolElement(StructurePoolElement, int)} for details.
+	 *
+	 * @param element the element to add
+	 */
+	void addStructurePoolElement(StructurePoolElement element);
+
+	/**
+	 * Adds a new {@link StructurePoolElement} to the {@link StructurePool}.
+	 *
+	 * @param element the element to add
+	 * @param weight  Minecraft handles weight by adding it that amount of times into a list.}
```

Comment:
```suggestion
	 * @param weight  Minecraft handles weight by adding it that amount of times into a list
```

Reply:
I'm not sure this is a good parameter description since this is an implementation detail.